zad4/app/services/football_player_manager.rb

```diff
@@ -8,14 +8,14 @@ def initialize(player)
   end
 
   def stronger_players
-    # TODO: Implement me pls
+    Player.where(power: @player.power..Float::INFINITY)
```

Comment:
Nie wiem czy zauważyłeś jakie zapytanie tu jest tworzone ;)
```sql
SELECT  "players".* FROM "players" WHERE "players"."power" >= ? LIMIT ?  [["power", 5], ["LIMIT", 11]]
```

Trochę też moja wina ze względu na nieścisłość polecenia, ale nie wiem czy Twoim zamiarem było wybranie większych i równych atrybutem `power` czy tylko większych :D Jak tylko większych to  `>=` może nie być oczekiwanym rezultatem :)

Ogółem można było to też czytelniej  bazodanowo zapisać:
```ruby
Player.where('power > ?', player.power)
```